src/com/sun/ts/tests/signaturetest/SigTestDriver.java

```diff
@@ -210,7 +210,7 @@ protected boolean runPackageSearch(String packageOrClassName,
   }
 
   /*
-   * @return This returns true if jakarta.transaction.xa is not found in the
+   * @return This returns true if javax.transaction.xa is not found in the
    * JTA API jar
    */
```

Comment:
We are going to regenerate/update the signaturetest folder changes separately,  so this changes in signaturetest aren't needed.

Reply:
Reverted all .java in src/com/sun/ts/tests/signaturetest back to master versions.